### PR TITLE
[AuthV2] BREAKING CHANGE: `az webapp auth-classic update`: Remove json output for auth-classic update

### PR DESCRIPTION
**Pull request by amberwang113**
---
Necessary to remove/redact output for auth-classic update command because secrets are getting unintentionally leaked in CI runs where users with higher privilege run update commands. Users should use "show" to get values of secrets instead of the returned output from update command. This change removes output from the command and fixes tests so that they're no longer reliant on the problematic behavior. 

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
az webapp auth-classic update 

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AuthV2

**Comment by amberwang113** [comment]
there are some style issues, but they all predate this PR

**Comment by yanzhudd** [comment]
Please add changlog to HISTORY.rst file. If you want to release a new version for this extension, please change setup.py as well.

**Comment by amberwang113** [comment]
Added version 0.1.3 to HISTORY.rst and changed version number in setup.py! @yanzhudd

**Review by yanzhudd — APPROVED** [review]

File: src/authV2/azext_authV2/tests/latest/test_authV2_scenario.py [hunk 1]
```diff
@@ -131,23 +131,25 @@ def test_authV2_authclassic(self, resource_group):
...
-                              JMESPathCheck('enabled', True),
-                              JMESPathCheck('tokenStoreEnabled', False),
-                              JMESPathCheck('tokenRefreshExtensionHours', 7.2),
-                              JMESPathCheck('runtimeVersion', '1.2.8'),
-                              JMESPathCheck('clientId', 'aad_client_id'),
-                              JMESPathCheck('clientSecretCertificateThumbprint', 'aad_thumbprint'),
-                              JMESPathCheck('issuer', 'https://issuer_url'),
-                              JMESPathCheck('facebookAppId', 'facebook_id')]).get_output_in_json()
+        # update
+        self.cmd('webapp auth-classic update -g {} -n {} --enabled true --action LoginWithFacebook '
```
Comment (yanzhudd) [inline src/authV2/azext_authV2/tests/latest/test_authV2_scenario.py:135]: It is suggested to add a check for the existence of output

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**